sphere-models/src/it/java/io/sphere/sdk/categories/queries/CategoryPredicatesTest.java

```diff
@@ -21,7 +21,7 @@
 public class CategoryPredicatesTest extends IntegrationTest {
     @Test
     public void isLessThanComparisonPredicate() throws Exception {
-        final QueryPredicate<Category> predicate = CategoryQuery.model().name().lang(Locale.ENGLISH).isLessThan("2");
+        final QueryPredicate<Category> predicate = CategoryQueryModel.of().name().lang(Locale.ENGLISH).isLessThan("2");
```

Comment:
It is better to keep is with the value shape since I use it as parameter to a function.